data/backend/reinit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rm -f db.sqlite3
+rm -f textile/migrations/*
+rm -f authentication/migrations/*
+python manage.py makemigrations mailauth authentication textile
```

Comment:
Why not just `python manage.py makemigrations` so we always generate them for all installed apps?

Reply:
It was a temporary setup during dev, it's being removed in the other PR with Db